Gemfile.lock

```diff
@@ -271,17 +271,17 @@ GEM
       railties (>= 5.0.0)
     faker (2.15.1)
       i18n (>= 1.6, < 2)
-    faraday (1.1.0)
+    faraday (1.2.0)
```

Comment:
`faraday` is used by a bunch of clients (Mailchimp, Buffer, Elasticsearch, Omniauth, GitHub).

According to the changelog - https://github.com/lostisland/faraday/releases/tag/v1.2.0 - seems to just be adding new features and fixing bugs, no breaking changes